### PR TITLE
Feature/team sidebar integration

### PR DESCRIPTION
**Pull request by Alexander-Lindstroem**

### CODE REVIEW

File: scripts/team.js [hunk 1]
```diff
@@ -31,14 +31,4 @@ const constructSidebarList = () => {
...
-
-/*<li>
-<a href="#">
-    <img src="../icons/echidna.png" alt="echidna Icon" class="icon-side echidna-icon">
-    <span class="nav-item">Echidna</span>
-</a>
-<span class="tooltip">Echidna</span>
-</li>*/
\ No newline at end of file
+constructSidebarList();
```
Comment (Alibaba000101) [inline scripts/team.js:33]: LGTM

**Review by Alibaba000101 — APPROVED** [review]
LGTM

File: html/team.html [hunk 2]
```diff
@@ -8,9 +8,18 @@
...
-    
+    <div class="sidebar">
+        <div class="top">
+            <div class="logo">
+                <img src="../icons/paw.png" alt="paw" class="iconpaw" id="btn">
+                <ul class="sidebar__list">
+                </ul>
+            </div>
+        </div>
+    </div>
```
Comment (DimitarIM) [inline html/team.html:33]: Icon stretches out weirdly when the sidebar opens up but overall LGTM

**Review by DimitarIM — APPROVED** [review]

File: html/team.html (hunk 2 above)
Comment (Alexander-Lindstroem) [reply html/team.html:33]: Fixed

**Pull request merged**